### DISCUSSION TITLE
How to Update

### DISCUSSION THREAD

**Discussion by hoeci** [Delusoire/bespoke#5] [Q&A] [answered]
Hello, Thanks for working hard on this project.
You are constantly updating the bespoke, I don't know how to update it. I try using `git pull`, but some files still not updated.
Since you didn't provide a way to update, I'm asking here. sorry, I'm new.

**Comment by Delusoire** [comment, marked as answer]
`git pull` isn't enough because this repository takes advantage of git submodules, which are basically repos embedded in other repos. You should instead run `git pull --recurse-submodules`.
